website/docs/r/api_management.html.markdown

```diff
@@ -163,6 +165,12 @@ A `proxy` block supports the following:
 
 ---
 
+A `protocols` block supports the following:
+
+* `enable_http2` - (Optional) Should HTTP/2 support be enabled on the client-facing side of the gateway? Defaults to `false`.
```

Comment:
just double-checking here since [the key is `Server.Http2`](https://github.com/terraform-providers/terraform-provider-azurerm/pull/5593/files#diff-6c4c0fbff8f080a23c6180cb0ab1e6d0R31)?

Reply:
Hi @tombuildsstuff, 
thanks for your feedback. So your suggestion is to rename `enable_http2` to `server_http2` ?

Reply:
as discussed offline, we should be able to update the documentation slightly here - but this is otherwise fine as-is

Reply:
Updated description to: Should HTTP/2 be supported by the API Management Service? Defaults to `false`.